docs/variants/msi_z690/installation-manual.md

```diff
@@ -56,6 +56,42 @@ backup or alternatively upload it to some cloud or network drive (Ubuntu live
 has a Firefox browser installed). Ubuntu live image is volatile and has no
 persistent storage. All changes made in live image will be lost after reboot.
 
+### Migrating SMBIOS unique data
+
+To migrate the SMBIOS system UUID and board serial number follow the Linux
+instructions below before attempting to flash the binary. The procedure is
+supported on Dasharo version v1.0.0 and later and requires cbfstool built from
+coreboot tree.
+
+```bash
+echo -n `sudo dmidecode -s system-uuid` > system_uuid.txt
+echo -n `sudo dmidecode -s baseboard-serial-number` > serial_number.txt
+# assuming in coreboot root directory
+./build/cbfstool build/coreboot.rom add -f serial_number.txt -n serial_number -t raw -r FW_MAIN_A
+./build/cbfstool build/coreboot.rom add -f serial_number.txt -n serial_number -t raw -r FW_MAIN_B
+./build/cbfstool build/coreboot.rom add -f serial_number.txt -n serial_number -t raw -r COREBOOT
+./build/cbfstool build/coreboot.rom add -f system_uuid.txt -n system_uuid -t raw -r FW_MAIN_A
+./build/cbfstool build/coreboot.rom add -f system_uuid.txt -n system_uuid -t raw -r FW_MAIN_B
+./build/cbfstool build/coreboot.rom add -f system_uuid.txt -n system_uuid -t raw -r COREBOOT
+```
+
+One may use `msi_ms7d25_v1.0.0.rom` binary directly and simply build the
+cbfstool only from coreboot repository:
+
+```bash
+cd coreboot
+make -C util/cbfstool
```

Comment:
We use it abov and build it after?

Reply:
In case someone haven't built binary from scratch one has to build cbfstool manually after cloning coreboot. Intructions above assume the binary has been built.

Reply:
Explained tsi section in more details